dasharo-performance/core-performance-boost.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+
+Suite Setup         Prepare Test Suite
+Suite Teardown      Log Out And Close Connection
+
+
+*** Test Cases ***
+CPB001.001 Check if disabling CPB decrease performance
+    Skip If    not ${CPB_SUPPORT}    CBP not supported
```

Comment:
These options are all part of APU menu, so maybe we ca reuse the single flag as here: https://github.com/Dasharo/open-source-firmware-validation/pull/221/files#diff-673a70ee5ebe88c297daaa3c52dc4d6e47b8440b7ab1b02489daaed919a15f20R33 ?

Maybe these should even be integrated into a single test suite.

Reply:
Yeah that's what I had in mind when I created [that file](https://github.com/Dasharo/open-source-firmware-validation/pull/221/files#diff-673a70ee5ebe88c297daaa3c52dc4d6e47b8440b7ab1b02489daaed919a15f20R33) , that all of the APU specific tests would go there.

Reply:
Done.